### ISSUE TITLE
Liquid Exception: HTTP Response: 503 Service Unavailable

### ISSUE THREAD

**Issue by scottcwilson** [tokzk/jekyll-amazon#7]
Suddenly I am getting 

/Users/scott/.rvm/gems/ruby-2.3.3/gems/amazon-ecs-2.4.0/lib/amazon/ecs.rb:141:in `send_request': HTTP Response: 503 Service Unavailable (Amazon::RequestError)

Is anyone else seeing this?  What's the fix?

**Comment by tokzk** [comment]
Amazon probably changed the access limitation of the API.

- https://forums.aws.amazon.com/thread.jspa?threadID=298518&tstart=25
- https://forums.aws.amazon.com/thread.jspa?messageID=894986&tstart=0

**Comment by scottcwilson** [comment]
Yep, I think that is exactly what happened.  I wonder if I could buy access to this API; I'd be happy to pay for it, I just don't want my code to stop working.
